generateMetadataJsonFromScriptResources.js

```diff
@@ -13,13 +13,17 @@ const resourcesDir = "resources";
 console.debug(`Creating ${metadataJsonFile}...`);
 const resources = fs.readdirSync(resourcesDir);
 const metadataList = resources.map((file) => {
+  const name = path.basename(file, ".js");
+  if (name.startsWith(".")) {
+    return;
+  }
```

Comment:
This seems like it could become a source of confusion eventually - would it be possible to keep lint configuration in the root directory instead?

Reply:
As mentioned below, Brave uses standardjs in general IIRC, which has the added benefit of not needed additional lint configuration.  Can we use that instead (and then get rid of this check too)

Reply:
you could also get rid of the confusing `filter` call too.  Why not use a simple for loop and not have to iterate over everything twice?